Source/monster.cpp

```diff
@@ -3897,10 +3895,9 @@ void InitMonsters()
 void SetMapMonsters(const uint16_t *dunData, Point startPosition)
 {
 	AddMonsterType(MT_GOLEM, PLACE_SPECIAL);
-	AddMonster(GolemHoldingCell, DIR_S, 0, false);
-	AddMonster(GolemHoldingCell, DIR_S, 0, false);
-	AddMonster(GolemHoldingCell, DIR_S, 0, false);
-	AddMonster(GolemHoldingCell, DIR_S, 0, false);
+	for (int k = 0; k < MAX_PLRS; k++)

```

Comment:
The use of `j` and then `k` to represent the same thing in the same commit looks weird to me.

Reply:
Fixed